frontend/packages/operator-lifecycle-manager/locales/en/olm.json

```diff
@@ -17,5 +121,57 @@
   "Uninstall Operator?": "Uninstall Operator?",
   "This will remove Operator <1>{{name}}</1> from <4>{{namespace}}</4>. Removing the Operator will not remove any of its custom resource definitions or managed resources. If your Operator has deployed applications on the cluster or configured off-cluster resources, these will continue to run and need to be cleaned up manually.": "This will remove Operator <1>{{name}}</1> from <4>{{namespace}}</4>. Removing the Operator will not remove any of its custom resource definitions or managed resources. If your Operator has deployed applications on the cluster or configured off-cluster resources, these will continue to run and need to be cleaned up manually.",
   "Message from Operator developer": "Message from Operator developer",
-  "Uninstall": "Uninstall"
+  "Uninstall": "Uninstall",
+  "The Operator Lifecycle Manager will not watch this Namespace because it is not configured with an OperatorGroup.": "The Operator Lifecycle Manager will not watch this Namespace because it is not configured with an OperatorGroup.",
+  "Create one here.": "Create one here.",
+  "Select OperatorGroup": "Select OperatorGroup",
+  "View Installed Operators in {namespace ? (\n          <>\n            namespace <b>{namespace}</b>\n          </>\n        ) : (\n          <b>all namespaces</b>\n        )}": "View Installed Operators in {namespace ? (\n          <>\n            namespace <b>{namespace}</b>\n          </>\n        ) : (\n          <b>all namespaces</b>\n        )}",
```

Comment:
It looks like we have another message with logic in it. We should try to clean that up.